appserver/jdbc/jdbc-ra/pom.xml

```diff
@@ -89,15 +89,15 @@
       <url>http://blogs.sun.com/JagadishPrasath</url>
       <organization>Oracle, Inc.</organization>
       <roles>
-        <role>lead</role>      
+        <role>lead</role>
 	<role>developer</role>
       </roles>
     </developer>
   </developers>
 
   <modules>
     <module>jdbc-core</module>
-    <module>jdbc30</module>
+    <!--<module>jdbc30</module>-->
```

Comment:
we can remove it instead of commenting it out.

Reply:
Done.